src/Illuminate/View/Compilers/Concerns/CompilesConditionals.php

```diff
@@ -119,4 +119,25 @@ protected function compileEndAuth()
     {
         return '<?php endif; ?>';
     }
+ 
+    /**
+     * Compile the if-guest statements into valid PHP.
+     *
+     * @param  string|null  $guard
+     * @return string
+     */
+    protected function compileGuest($guard = null)
+    {
+        return "<?php if(auth()->guard{$guard}->guest()): ?>";
```

Comment:
No guys!
This is not the guard which is passed, but the complete $expression, with the ().
Nothing to change here!